pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -712,6 +723,42 @@
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+ ::  +handle-cache-req: respond with cached value, 404 or 500
+  ++  handle-cache-req
+    |=  [authenticated=? =request:http =path]
+    |^  ^-  (quip move server-state)
+    ?.  authenticated
+      (error-response 403 ~)
+    ?.  =(%'GET' method.request)
+      (error-response 405 "may only GET scries")
+    =/  cache-entry  (~(get of cache.state) path)
+    ?~  cache-entry
+      (error-response 404 "no cache entry for that binding")
+    ?-    -.u.cache-entry
+        %payload
+      =/  payload  simple-payload.u.cache-entry
```

Comment:
`=*  payload  simple-payload.u.cache-entry`

Reply:
ah, I think I see the rule -- use `=/` when computing new values, and `=*` when referencing existing values. Is that accurate?

Reply:
That's right!